.github/workflows/stage-4-acceptance.yaml

```diff
@@ -60,17 +60,39 @@ jobs:
           role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.AWS_ASSUME_ROLE_NAME }}
           role-session-name: deployInfra
           aws-region: ${{ env.AWS_REGION }}
-      - name: "Create sandbox"
+      - name: "Create Amplify sandbox"
         run: |
-          ./scripts/create_sandbox.sh
+          ./scripts/create_amplify_sandbox.sh
       - uses: actions/upload-artifact@v4
         with:
           name: amplify_outputs.json
           path: amplify_outputs.json
+  sandbox-set-up:
```

Comment:
I dont believe this will work. under the AMET deigns there is no ability for the component/microservice repositories to be able to do anything with the NHS Notify deployment context. The internal repo is where we manage how the services are deployed in the context of NHS Notify as the service.

@mark-r-bjss @timireland @RossBugginsNHS @m-houston and I discussed a mechanism by where there would/could be a workflow in the internal side of the programme whereby this sort of targeted creation and execution of service context testing could be run; though I don't know what happened there in my absence over the last few weeks

basically the service repos cant access AWS so this aint gunna work

Reply:
spoke to Mark R, this is a workaround